ossec_security/assets/logs/ossec-security.yaml

```
@@ -0,0 +1,708 @@
+              system OR ftpd)
+          name: other
+      target: service
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: pipeline
+      name: Processing of Firewall Logs
+      enabled: true
+      filter:
+        query: "@rule_id:([4100 TO 4299] OR [40900 TO 40999]) "
+      processors:
+        - type: grok-parser
+          name: Parse firewall logs.
+          enabled: true
+          source: log
+          samples:
+            - 2006-09-20 10:38:21 DROP TCP 10.10.10.10 10.10.10.10 1000 1000 48
+              SA 4175551841 892874455 17520 - - - RECEIVE
+            - >
+              2006-09-19 03:31:15 DROP UDP 10.10.10.10 10.10.10.10 1000 1000 310
+              - - - - - - - RECEIVE
+            - 2006-09-19 03:27:05 CLOSE UDP 11.11.11.11 11.11.11.11 3682 88 - -
+              - - - - - - -
+            - 2006-09-19 03:37:57 OPEN TCP 11.11.11.11 11.11.11.11 3686 80 - - -
+              - - - - - -
+            - 2006-09-19 04:07:56 DROP UDP 12.12.12.12 12.12.12.12 2250 1900 250
+              - - - - - - - RECEIVE
+          grok:
+            supportRules: ""
+            matchRules: firewall_parsing_rule %{date("yyyy-MM-d HH:mm:ss"):} %{word:action}
+              %{word:protocol} %{ip:src_ip} %{ip:dst_ip} %{integer:src_port}
+              %{integer:dst_port} %{regex("[0-9-]*"):size:nullIf("-")}
+              %{regex("[A-Za-z0-9-]*"):tcpflags:nullIf("-")}
+              %{regex("[0-9-]*"):tcpsyn:nullIf("-")}
+              %{regex("[0-9-]*"):tcpack:nullIf("-")}
+              %{regex("[0-9-]*"):tcpwin:nullIf("-")}
+              %{notSpace:icmptype:nullIf("-")} %{notSpace:icmpcode:nullIf("-")}
+              %{notSpace:info:nullIf("-")} %{notSpace:path:nullIf("-")}
+        - type: attribute-remapper
+          name: Map `dst_ip` to `network.destination.ip`
+          enabled: true
+          sources:
+            - dst_ip
+          sourceType: attribute
+          target: network.destination.ip
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `dst_port` to `network.destination.port`
+          enabled: true
+          sources:
+            - dst_port
+          sourceType: attribute
+          target: network.destination.port
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `src_port` to `network.client.port`
+          enabled: true
+          sources:
+            - src_port
+          sourceType: attribute
+          target: network.client.port
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Processing of Adduser Logs
```

```suggestion
      name: Process of adduser Logs
```
Just for consistency -- if these edits will cause any programmatic issues, you can ignore them!

---

Updated as per suggestion